### DISCUSSION TITLE
Does anybody know how I can enable tiling for floating windows?

### DISCUSSION THREAD

**Discussion by gerber4** [i3/i3#4875] [Q&A]
I understand this is a confusing question. What I mean is tiling similar to how it is done in Windows. When you drag a window to the edge of a monitor is will expand to fill half of the screen. Also, I would want to support hotkeys that tile floating windows. Are there any tools that would help me get this behavior?

**Comment by bensnowball** [comment]
There is no "snap-to" (I think that's what it's called on Windows) behavior for floating windows. But there is an option for toggling between tiling or floating for the currently focused window [here](https://i3wm.org/docs/userguide.html#_floating), the default key for it is Mod+Shift+Space.
